ddtrace/contrib/botocore/services/sqs.py

```diff
@@ -100,8 +100,7 @@ def inject_trace_to_sqs_or_sns_message(params, span, endpoint_service=None):
     """
     trace_data = {}
     HTTPPropagator.inject(span.context, trace_data)
-
-    core.dispatch("botocore.sqs_sns.start", [endpoint_service, trace_data, params])
+    core.dispatch("botocore.sqs_sns.start", [endpoint_service, trace_data, params, params])
```

Comment:
I don't understand this `params, params` here 🤔 
If it's the right thing to do, maybe a comment would be nice ?

Reply:
Simplified this by removing the second use of params. This was originally done since params represents both the top level parameters used to call the function, as well as the message object containing all message data. For batch operations, these two are not the same, so we needed both separately.